tests/testthat/test-processCapabilityStudies.R

```diff
@@ -1 +1,97 @@
-context("Process Capability Studies")
+context("[Quality Control] Process Capability Studies")
+
+options <- analysisOptions("processCapabilityStudies")
+options$capabilityStudy <- "initialCapabilityAnalysis"
+options$lowerSpecification <- 4
+options$lowerSpecificationField <- TRUE
+options$nonNormalCapabilityStudy <- TRUE
+options$nullDistribution <- "Lognormal"
+options$rank <- "Bernard"
+options$targetValue <- 8
+options$targetValueField <- TRUE
+options$upperSpecification <- 12
+options$upperSpecificationField <- TRUE
+options$variables <- c("Measurement1", "Measurement2", "Measurement3")
+set.seed(1)
+results <- runAnalysis("processCapabilityStudies", "C:/Users/derksk/OneDrive - NBU/Desktop/PhD/JASP modules/jaspProcessControl/tests/testthat/partOperatorData.csv", options)
```

Comment:
well, that won't work, and if it does it's some kind of voodoo black magic. Shouldn't you add the test file to the tests folder and then reference it directly?

Reply:
Oops